src/palakSJun20/BankingSystem.java

```diff
@@ -0,0 +1,85 @@
+/*Create a Banking System which has following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output:
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variable will be required.
+Output:
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+package palakSJun20;
+
+public class BankingSystem {
+	String customerName;
+	int customerCredit , customerDebit , customerBalance;
+	int customerCreditCount , customerDebitCount , customerPrintBalanceCount;
+	static int creditCount, debitCount, printBalanceCount;
+	
+	void setCustomerName(String customerName){
+		this.customerName = customerName;
+	}
+	
+	int debitTransaction(int debitAmount){
+		customerDebitCount++;
+		debitCount++;
+		return this.customerBalance = customerBalance - debitAmount;
```

Comment:
You should write the condition to verify the insufficient balance.

Reply:
Added condition